src/main/java/com/books/storage/abstracts/Repository.java

```diff
@@ -0,0 +1,16 @@
+package com.books.storage.abstracts;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface Repository<T> {
+    // void or boolean?
```

Comment:
Why we need to use boolean? I think better to raise question: void or saved object :)